app/graph/types/feed_type.rb

```diff
@@ -5,13 +5,46 @@
   interfaces [NodeIdentification.interface]
 
   field :dbid, types.Int
+  field :user_id, types.Int
   field :name, types.String
+  field :description, types.String
   field :published, types.Boolean
   field :filters, JsonStringType
   field :current_feed_team, FeedTeamType
   field :teams_count, types.Int
   field :requests_count, types.Int
   field :root_requests_count, types.Int
+  field :tags, JsonStringType
+  field :licenses, types[types.Int]
+  field :saved_search_id, types.Int
+  field :team_id, types.Int
+
+  field :user do
+    type -> { UserType }
+
+    resolve -> (feed, _args, ctx) {
+      RecordLoader.for(User).load(feed.user_id).then do |user|
+        ability = ctx[:ability] || Ability.new
+        user if ability.can?(:read, user)
+      end
+    }
+  end
+
+  field :team do
+    type -> { TeamType }
+
+    resolve -> (feed, _args, _ctx) {
+      RecordLoader.for(Team).load(feed.team_id)
+    }
+  end
+
+  field :saved_search do
+    type -> { SavedSearchType }
+
+    resolve -> (feed, _args, _ctx) {
+      RecordLoader.for(SavedSearch).load(feed.saved_search_id)
```

Comment:
Suggestion: No need for `RecordLoader` here.

Reply:
Done